Cargo.toml

```diff
@@ -72,11 +72,14 @@ serde = { version = "1.0.164", features = ["derive"] }
 toml = "0.8.13"
 home = "0.5.9"
 which = "6.0.1"
+ratatui = "0.28"
```

Comment:
You can also gate `ratatui` dependency behind `binsider-integration` feature I think